src/app/debug/Debug.tsx

```diff
@@ -0,0 +1,321 @@
+
+        <section>
+          <h3>Accounts</h3>
+          {accounts.map((account) => (
+            <div key={`${account.membershipId}-${account.destinyVersion}`}>
+              <Account account={account} selected={account === currentAccount} />
+              {account.membershipId}
+            </div>
+          ))}
+        </section>
+
+        <section>
+          <h3>Storage</h3>
+          <p>
+            {localStorageError ? (
+              <>
+                <b>Local Storage Broken:</b> <ErrorInfo error={localStorageError} />
+              </>
+            ) : (
+              <b>Local Storage Works</b>
+            )}
+          </p>
+          <p>
+            {idbError ? (
+              <>
+                <b>IDB Broken:</b> <ErrorInfo error={idbError} />
+              </>
+            ) : (
+              <b>IDB Works</b>
+            )}
+          </p>
+          <LocalStorageInfo showDetails />
+        </section>
+
+        <section>
+          <h3>Profile Response</h3>
+          {now}
+          {profileResponse && (
+            <p>
+              <b>Minted:</b> {new Date(profileResponse?.responseMintedTimestamp).toISOString()}
+            </p>
+          )}
+          <p>
+            <b>Error:</b> {profileError ? <ErrorInfo error={profileError} /> : 'None'}
+          </p>
+        </section>
+
+        <section>
+          <h3>DIM Sync</h3>
+          <p>
+            <b>API Permission Granted:</b> {JSON.stringify(apiPermissionGranted)}
+          </p>
+          {now}
+          {dimSyncProfile ? (
+            <>
+              <p>
+                <b>Last Fetched:</b> {new Date(dimSyncProfile.profileLastLoaded).toISOString()}
+              </p>
+              <p>
+                <b>Tags:</b> {Object.keys(dimSyncProfile.tags).length}
+              </p>
+              <p>
+                <b>Loadouts:</b> {Object.keys(dimSyncProfile.loadouts).length}
+              </p>
+            </>
+          ) : (
+            <p>No Profile Loaded</p>
+          )}
+          <p>
+            <b>Update Queue Size:</b> {dimSyncUpdateQueueSize}
+          </p>
+          <p>
+            <b>Error:</b> {dimSyncError ? <ErrorInfo error={dimSyncError} /> : 'None'}
+          </p>
+        </section>
+
+        <section>
+          <h3>Service Worker</h3>
+          {serviceWorkers.length > 0 ? (
+            serviceWorkers.map((w, i) => (
+              <div key={i}>
+                {i}: Active: {w.active?.state || 'null'}, Waiting: {w.waiting?.state || 'null'},
+                Installing: {w.installing?.state || 'null'}
+              </div>
+            ))
+          ) : (
+            <div>Service worker not installed</div>
+          )}
+        </section>
+
+        <section>
+          <h3>Wish Lists</h3>
+          <p>
+            <b>Source:</b> {wishlistSource}
+          </p>
+          <p>
+            <b>Last Fetched:</b> {wishListsLastFetched?.toISOString() ?? 'Never'}
+          </p>
+        </section>
```

Comment:
One candidate here would be something to debug #7723 / #9467 and figure out why sometimes `recommendedPerks` seems to be not null and not undefined but also not iterable.